src/patterns/ec2-app.ts

```diff
@@ -48,6 +50,10 @@ interface RestrictedAccess extends Access {
 
 export type AppAccess = PublicAccess | RestrictedAccess;
 
+export interface AccessLoggingProps {
+  prefix: string | false;
```

Comment:
Now that we are using a single bucket per account I don't think `false` is going to be a sensible choice here (more generally, I am not super keen on odd union types like this). Perhaps we can provide a sensible default (`stack/stage/app`?) or just make the user give us a `string`?

Reply:
Hmm, yeah, I think that's a sensible suggestion. To unblock ourselves, I'm going to add a `TODO` and merge this in so we can address this later.

Reply:
On second thought, I think you're right in having the user give us a string. I've implemented this.

Reply:
Agreed, using the presence of a string as a boolean proxy isn't ideal and it's better to be explicit, would something like this work:

```typescript
interface AccessLoggingProps {
  enabled: boolean;
  prefix?: string;
}
```

This has another benefit or making it easy to explicitly disable access logging, rather than relying on it being the default behaviour.